### DISCUSSION TITLE
Do you have an android version?

### DISCUSSION THREAD

**Discussion by garawaa** [fairy-stockfish/Fairy-Stockfish#247] [Q&A]
Do you have an android version?

**Comment by ianfab** [comment]
I do not have any experience with development for Android, but since this project is based on https://github.com/official-stockfish/Stockfish, compilation for Android should work the same, e.g., as documented in https://github.com/official-stockfish/Stockfish/commit/65b976439f8867e81682c0b66da6796ad3176177.
